loader/src/loader.rs

```diff
@@ -83,7 +84,7 @@ struct AssetVersionLoad {
 #[derive(Debug)]
 struct AssetLoad {
     asset_id: AssetUuid,
-    last_state_change_instant: std::time::Instant,
```

Comment:
If `std::time` is banned, do you think I'll have any issues with my work on removing `tokio`? I'm wondering if any of the dependencies I'm using are expecting `std::time`.

Reply:
Honestly not sure, I think we'd have to try it and see. Won't know until it fails at runtime unfortunately.